sonar-batch/src/main/java/org/sonar/batch/bootstrap/BatchComponents.java

```diff
@@ -66,8 +67,16 @@ private BatchComponents() {
       HtmlReport.class,
       IssuesReportBuilder.class,
       SourceProvider.class,
-      RuleNameProvider.class
-      );
+      RuleNameProvider.class,
+
+      // Tasks
+      Tasks.class,
+      ListTask.DEFINITION,
+      ListTask.class,
+      ScanTask.DEFINITION,
+      ScanTask.class,
+      ViewsTask.DEFINITION,
+      ViewsTask.class);
```

Comment:
In any case, I think failing fast on broken components is good. That's one of the good things about medium tests.